src/main/flight/pid.c

```diff
@@ -266,6 +268,8 @@ void pidInitFilters(const pidProfile_t *pidProfile)
         ptermYawLowpassApplyFn = (filterApplyFnPtr)pt1FilterApply;
         pt1FilterInit(&ptermYawLowpass, pt1FilterGain(pidProfile->yaw_lowpass_hz, dT));
     }
+
+    pt1FilterInit( &throttlelpf, pt1FilterGain( pidProfile->throttle_boost_cutoff, targetPidLooptime * 0.000001f ));
```

Comment:
`dT` is used for other filters (inconsistently)

Reply:
nice, yes, that's better.